src/enums.js

```diff
@@ -74,6 +74,15 @@ const MojaloopTypes = {
   }
 }
 
+/**
+ *  Mojaloop API Error Codes Override
+ */
+const MojaloopApiErrorCodesOverride = {
```

Comment:
I am not sure I understand what you are overriding here. Are you overriding existing "mojaloop" error codes, or are you adding new error codes to cater for non-ML specific error scenarios?

I.e. The httpStatusCodes that you are setting below should be aligned to the spec. If they are, then this should be changed in the SDK.

If we need different httpStatusCode's for async vs sync requests (i.e. non-ML API calls), then we should probably add a new custom error.

Let me know what you think?

Reply:
The proposed change overrides existing error codes. I see your point and I think we should be able to add new custom errors and only be able to extend the existing ones for what hasn't been set already by the SDK. I'm ready to make the change if you agree?

Reply:
This has been resolved as discussed earlier today. Please, review latest changes.